core/pyproject.toml

```diff
@@ -62,12 +64,15 @@ parquet = ["fastparquet"]
 pg = ["psycopg2-binary", "SQLAlchemy"]
 redis = ["redis"]
 sqlserver = ["pymssql", "SQLAlchemy"]
+kafka = ["kafka-python", "confluent-kafka"]
 
 test = [
         "aiohttp",
         "cassandra-driver",
+        "confluent-kafka",
         "fastparquet",
         "ibm_db_sa",
+        "kafka-python",
```

Comment:
you don't need to specify both `confluent-kafka` and `"kafka-python`. Just put `kafka` and it will install the packages listed in line 67.

Reply:
Ok